js/layout-javascript/agenda-code.js

```diff
@@ -1227,6 +1233,8 @@ DynamicList.prototype.addSummaryData = function(records) {
         content = _this.convertTime(content);
       }
 
+      content = _this.Utils.DataSource.formatCellValue(content);
```

Comment:
```suggestion
      content = _this.Utils.String.toFormattedString(content);
```

`_this.Utils.String.toFormattedString()` should:

1. If the input is a string, return the string
2. If the input is undefined or null, return an empty string
3. If the input is an array, remove the following values and then return the value after join them with `, `
3. If the input is an object, return the value using `JSON.stringify()`

Reply:
@tonytlwu, fixed

Reply:
Point 2 specified:

> If the input is undefined or null, return an empty string

![image](https://user-images.githubusercontent.com/290733/106655471-4f4c5d00-6591-11eb-9fcb-aeb3ae950181.png)

Reply:
@tonytlwu, fixed
![изображение](https://user-images.githubusercontent.com/71251809/106660886-3c458700-65a9-11eb-9ddc-2d8a4845a346.png)